dataland-frontend/src/components/forms/parts/fields/NumberFormField.vue

```diff
@@ -1,17 +1,21 @@
 <template>
   <div class="mb-3">
-    <UploadFormHeader :label="label" :description="description ?? ''" :is-required="required" />
+    <UploadFormHeader :label="label" :description="description" :is-required="required" />
     <div class="next-to-each-other">
       <FormKit
         type="text"
         :name="name"
         :unit="unit"
+        :value="currentValue"
         :validation-label="validationLabel ?? label"
         :validation="`number|${validation}`"
         :placeholder="unit ? `Value in ${unit}` : 'Value'"
-        outer-class="short"
+        :validationMessages="{ integer: `${validationLabel ?? label} must be an integer.` }"
```

Comment:
Why does a NumberFormField have validation rules for integers hardcoded?

Reply:
The rule to check is a value is an integer we need in the numberFieldForm component, if we want to us it as a common component for all extended datapoints. The validationMessage however we could probably change to a variable and pass it on from the IntegerDataPoint component. What do you think?

Reply:
And the rules only get applied if the `validation` attribute contains `integer`.